pypeit/core/skysub.py

```diff
@@ -1471,6 +1471,8 @@ def generate_mask(pypeline, skyreg, slits, slits_left, slits_right, spat_flexure
 
     # Now that we have sky region traces, utilise the SlitTraceSet to define the regions.
     # We will then use the slit_img task to create a mask of the sky regions.
+    # TODO: Is the datatype correct here?  It should be set by the SlitTraceSet
```

Comment:
You're right... I think this should just be `np.integer` - I see no harm in changing it  :-)